apps/bfd-server/bfd-server-war/src/test/resources/openapi/posts.yaml

```diff
@@ -354,3 +354,241 @@ paths:
+                      represents the medicare benficiary ID, or as much of that
+                      defined by the [FHIR specification](https://terminology.hl7.org/NamingSystem-cmsMBI.html) as:
+
+                      ```
+                      A combination of upper case ASCII letters ('A'..'Z except for S, L, O, I, B, and Z.', numerals ('0'..'9'),
+                      '-', with a length limit of 11 characters.
+                      ```
+
+                      The following are all valid values for MBI, and all might
+                      represent the same resource:
+                        - `mbi=9AB2WW3GR44` (unhashed MBI)
+                        - `mbi=82273caf4d2c3b5a8340190ae3575950957ce469e593efd7736d60c3b39d253c` (hashed)
+                        - `mbi=1S00E00HA26` (synthetic MBI)
+                    example: "9AB2WW3GR44"
+                  _lastUpdated:
+                    type: string
+                    description: |-
+                      Only satisfy the Search if the Beneficiary's `last_updated` Date falls within a specified _DateRange_.
+                      A _DateRange_ can be defined by providing less than `lt` and/or greater than `gt` values.
+                      This parameter can be included in a request one or more times.
+
+                      Examples:
+                        - `_lastUpdated=gt2023-01-02&_lastUpdated=lt2023-05-01` defines a range between two provided dates
+                        - `_lastUpdated=gt2023-01-02` defines a range between the provided date and today
+                        - `_lastUpdated=lt2023-05-01` defines a range from the earliest available records until the provided date
+                    example: "gt2023-01-02"
+                  excludeSAMHSA:
+                    type: string
+                    description: |-
+                      The _Substance Abuse and Mental Health Services Administration_ (SAMHSA)
+                      is the agency within the U.S. Department of HHS that leads public health efforts to advance the behavioral health of the nation.
+                      Setting this flag to _true_, modifies the request to filter out all SAMSHA-related claims from the response.
+                      _ClaimResponse_ doesn't contain any SAMHSA potential data. Due to the 1:1 relationship between _ClaimResponse_
+                      and _Claim_, any ClaimResponse resource will be excluded when the associated Claim resource contains SAMHSA data.
+
+                      Example:
+                        - `excludeSAMHSA=true`
+                    example: true
+                  identifier:
+                    type: string
+                    includeTaxNumbers: |-
+                      Setting this flag to _true_, provides tax number in the EOB transformed data for the response.
+
+                      Example:
+                        - `includeTaxNumbers=true`
+                    example: true
```

Comment:
Having another look at this... `identifier` and/or `includeTaxNumbers` don't look quite right.

Reply:
fixed